docs/native/src/overview.rst

```diff
@@ -1,5 +1,76 @@
 
 ..  _intro:
 
-|stub-icon| Integrating with Office for iOS
-===========================================
+Integrating with Office for iOS
+===============================
+You can integrate with Office for iOS to enable your users to view and edit Excel, PowerPoint, and Word files directly on the iPad and iPhone.
+
+.. image:: /images/ScenarioEdit.png
```

Comment:
For the image names, I suggest following the underscore naming convention used elsewhere (e.g. `scenario_edit.png`)